index.html

```diff
@@ -17,7 +17,7 @@
       gtag('config', 'G-KLG42VWE4W');
     </script>
 
-    <title>Portfolio</title>
+    <title>Fabrizio's Portfolio</title>
```

Comment:
## Title Changed
The title now briefly describes what this content is about, in this case, is a portfolio, but the "Fabrizio's Portfolio"